cli/cmd/generate_gcp.go

```diff
@@ -0,0 +1,774 @@
+	generateGcpTfCommand.PersistentFlags().BoolVar(
+		&GenerateGcpCommandState.OrganizationIntegration,
+		"organization_integration",
+		false,
+		"enable organization integration")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.GcpOrganizationId,
+		"organization_id",
+		"",
+		"specify the organization id (only set if organization_integration is set)")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.GcpProjectId,
+		"project_id",
+		"",
+		"specify the project id to be used to provision lacework resources")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpExistingServiceAccountDetails.Name,
+		"existing_service_account_name",
+		"",
+		"specify existing service account name")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpExistingServiceAccountDetails.PrivateKey,
+		"existing_service_account_private_key",
+		"",
+		"specify existing service account key (base64 encoded)")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.ConfigIntegrationName,
+		"config_integration_name",
+		"",
+		"specify a custom config integration name")
+	// TODO: Implement AuditLogLabels, BucketLabels, PubSubSubscriptionLabels & PubSubTopicLabels
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.BucketRegion,
+		"bucket_region",
+		"",
+		"specify gcp bucket region")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.BucketLocation,
+		"bucket_location",
+		"",
+		"specify gcp bucket location")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.BucketName,
+		"bucket_name",
+		"",
+		"specify new bucket name")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.ExistingLogBucketName,
+		"existing_bucket_name",
+		"",
+		"specify existing bucket name")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.ExistingLogSinkName,
+		"existing_sink_name",
+		"",
+		"specify existing sink name")
+	generateGcpTfCommand.PersistentFlags().BoolVar(
+		&GenerateGcpCommandState.EnableForceDestroyBucket,
+		"enable_force_destroy_bucket",
+		false,
+		"enable force bucket destroy")
+	generateGcpTfCommand.PersistentFlags().BoolVar(
+		&GenerateGcpCommandState.EnableUBLA,
+		"enable_ubla",
+		false,
+		"enable universal bucket level access(ubla)")
+	generateGcpTfCommand.PersistentFlags().IntVar(
+		&GenerateGcpCommandState.LogBucketLifecycleRuleAge,
+		"bucket_lifecycle_rule_age",
+		-1,
+		"specify the lifecycle rule age")
+	generateGcpTfCommand.PersistentFlags().IntVar(
+		&GenerateGcpCommandState.LogBucketRetentionDays,
+		"bucket_retention_days",
+		0,
+		"specify the bucket retention days")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.AuditLogIntegrationName,
+		"audit_log_integration_name",
+		"",
+		"specify a custom audit log integration name")
```

Comment:
I think that we need a flag to be able to run the terraform in non-interactive mode like "--terraform-apply" etc

Reply:
good catch. Will add this in